lua/core/plugins.lua

```diff
@@ -76,6 +79,7 @@ if packer_status_ok then
     -- Better buffer closing
     {
       "famiu/bufdelete.nvim",
+      event = "BufReadPost",
```

Comment:
This shouldn't be here. This is lazy loaded on the commands so it's already lazy loaded more aggressively than this